railties/test/application/configuration_test.rb

```diff
@@ -3124,7 +3124,7 @@ class ::DummySerializer < ActiveJob::Serializers::ObjectSerializer; end
     test "active record job queue is set" do
       app "development"
 
-      assert_equal ActiveSupport::InheritableOptions.new(destroy: :active_record_destroy), ActiveRecord.queues
```

Comment:
This was removed 3 years ago in #40766 by @rafaelfranca and has been false passing ever since due to `InheritableOptions` not behaving as it should with respect to equality and other hashes.